### PR TITLE
Fix NANs in pie chart

### PR DESCRIPTION
**Pull request by isaacroldan**
#### :tophat: What? Why?

Don't divide by 0!
#### :clipboard: Developer checklist
- [x] Unit Tests (if necessary)
- [x] UI Snapshot Tests (if necessary)
- [x] Added accessibility tags to any interactive element
- [x] Added reporting for GA (events + screen views) / Errors / Exceptions
- [x] Documentation

### CODE REVIEW

File: PNChart/PNPieChart.m [hunk 1]
```diff
@@ -124,7 +124,8 @@ - (UILabel *)descriptionLabelForItemAtIndex:(NSUInteger)index{
     UILabel *descriptionLabel = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, 100, 60)];
     NSString *titleText = currentDataItem.textDescription;
     
-    NSMutableAttributedString* attributedTitleText = [[NSMutableAttributedString alloc] initWithString:[NSString stringWithFormat:@"%.0f%%\n",currentDataItem.value/ _total * 100] attributes:@{NSForegroundColorAttributeName:currentDataItem.color,NSFontAttributeName:_fontForDetailItemNumber}];
+    int normalizedValue = (_total == 0 ? 0 : currentDataItem.value/ _total) * 100;
```
Comment (polqf) [inline PNChart/PNPieChart.m:127]: maybe clearer like this ? `_total == 0 ? 0 : (currentDataItem.value/ _total) * 100`

Comment (isaacroldan) [inline PNChart/PNPieChart.m:127]: the parenthesis are better to separate the first `=` to the next `==` that could be confusing

Comment (mariabernis) [inline PNChart/PNPieChart.m:127]: I wouldn't set the % to 0% when there is no value. I would print sth like "- %" or  another text to indicate "empty state". Maybe ask design or PO?

**Comment by mariabernis** [comment]
Ok to have it as in web. :+1:

File: PNChart/PNPieChart.m (hunk 1 above)
Comment (polqf) [inline PNChart/PNPieChart.m:127]: I'm saying that because it is useless 0*100, but anyway, maybe to many parenthesis then :joy:

**Comment by polqf** [comment]
:+1:

**Pull request merged**